mail_outbound_static/__manifest__.py

```diff
@@ -4,10 +4,13 @@
 {
     "name": "Mail Outbound Static",
     "summary": "Allows you to configure the from header for a mail server.",
-    "version": "13.0.2.0.0",
+    "version": "13.0.3.0.0",
```

Comment:
Shouldn't this rather be a minor update? `13.0.2.1.0`

I think the definition is 

- Requires restart only
- Requires update
- Full refactor, needs human attention to implement

But i'm not 100% sure

Reply:
Hm.. yes, I was not sure too. I've updated main version, because of implementation of reply logic, not sending. So it is "logical" up. But if you say so, i can make 2.1.0.
So - change?

Reply:
Leave it for now, lets await a third opinion.

Reply:
it's a minor update in my book too

Reply:
Corrected